lib/hex_web/package.ex

```diff
@@ -187,11 +187,11 @@ defmodule HexWeb.Package do
       name_search = "%" <> name_search <> "%"
     end
 
-    desc_search = String.replace(search, ~r"\s+", " & ")
+    desc_search = String.replace(search, ~r"\s+", " | ")
 
       from var in query,
     where: ilike(var.name, ^name_search) or
-           fragment("to_tsvector('english', (?->'description')::text) @@ to_tsquery('english', ?)",
+           fragment("to_tsvector('english', regexp_replace((?->'description')::text, '/', ' ')) @@ to_tsquery('english', ?)",
```

Comment:
> I'm not sure about this because it feels fragile to just change characters.

I agree.

> Is `/` the only character that will cause or will we have to keep tweaking this in the future?

Probably not.

With the `_` and `-` works by default:

```sql
-- package: rsa
select to_tsvector('...public_key...') @@ to_tsquery('public | key') as result;
-- result: true
select to_tsvector('...public_key...') @@ to_tsquery('public & key') as result;
-- result: true

-- package: benchwarmer
select to_tsvector('...micro-benchmarking...') @@ to_tsquery('micro | benchmarking') as result;
-- result: true
select to_tsvector('...micro-benchmarking...') @@ to_tsquery('micro & benchmarking') as result;
-- result: true
```

With the `.` character for example does not work on cases like this:

```sql
-- package: eio
select to_tsvector('...engine.io') @@ to_tsquery('engine | io') as result;
-- result: false
select to_tsvector('...engine.io') @@ to_tsquery('engine & io') as result;
-- result: false
```

But I really prefer not to include the `.` character in the `regexp_replace` function.